public/i18n/en.json

```diff
@@ -21,5 +21,7 @@
   "query-builder-value-type-relation-dropdown-regardless-of-value": "regardless of value",
   "query-builder-delete-condition": "Delete condition",
   "query-builder-limit-number-results-description": "Limit the number of results to",
+  "query-builder-limit-number-placeholder": "enter a number",
```

Comment:
Is there a specific reason for this to be not to be capitalized? ( First letter in capitals? )

Reply:
That is how it was in https://phabricator.wikimedia.org/T270712 :shrug:, but maybe that was not intentional. I'll ask UX.

Reply:
Done. It is now uppercase.